xbmc/windows/GUIWindowScreensaver.cpp

```diff
@@ -28,16 +28,20 @@ void CGUIWindowScreensaver::Process(unsigned int currentTime, CDirtyRegionList &
 {
   MarkDirtyRegion();
   CGUIWindow::Process(currentTime, regions);
-  m_renderRegion.SetRect(0, 0, (float)CServiceBroker::GetWinSystem()->GetGfxContext().GetWidth(), (float)CServiceBroker::GetWinSystem()->GetGfxContext().GetHeight());
+  auto& context = CServiceBroker::GetWinSystem()->GetGfxContext();
```

Comment:
`const auto&` :) please

Reply:
Have changed this line, as `GetWidth()` and `GetHeight()` defined as `const` it works. On the other two places not possible, as `ApplyStateBlock` and `CaptureStateBlock` do something.